lib/App/DH.pm

```diff
@@ -261,15 +261,31 @@ sub cmd_upgrade {
   return;
 }
 
+=cmd version
+
+Report version of connection L</--connection_name>
+
+    dh.pl [...params] version
+
+=cut
+
+sub cmd_version {
```

Comment:
This should probably add an entry in the `=begin Pod::Coverage` block later in the source to appease the pod coverage tests.